config/kubernetes/sig-node/teams.yaml

```diff
@@ -2,102 +2,112 @@ teams:
   sig-node-api-reviews:
     description: API Reviews for Kubernetes Node Special-Interest-Group
     members:
-    - ConnorDoyle
-    - davidopp
     - dchen1107
     - derekwaynecarr
-    - feiskyer
-    - resouer
+    - smarterclayton
     - tallclair
-    - vikaschoudhary16
-    - vishh
+    - thockin
     - yujuhong
     privacy: closed
   sig-node-bugs:
     description: Bugs for Kubernetes Node Special-Interest-Group
     members:
-    - ConnorDoyle
+    - andrewsykim
+    - bobbypage
     - dchen1107
     - derekwaynecarr
+    - dims
     - feiskyer
-    - resouer
-    - tallclair
-    - vikaschoudhary16
-    - vishh
-    - yguo0905
+    - klueska
+    - krmayankk
+    - matthyx
+    - mrunalp
+    - mtaufen
+    - odinuge
+    - pacoxu
+    - Random-Liu
+    - SergeyKanzhelev
+    - sjenning
+    - wzshiming
     - yujuhong
     privacy: closed
   sig-node-feature-requests:
     description: Feature Requests for Kubernetes Node Special-Interest-Group
     members:
-    - ConnorDoyle
-    - davidopp
+    - andrewsykim
+    - bobbypage
     - dchen1107
     - derekwaynecarr
+    - dims
     - feiskyer
+    - klueska
+    - krmayankk
+    - matthyx
+    - mrunalp
+    - mtaufen
+    - odinuge
+    - pacoxu
     - Random-Liu
-    - resouer
-    - smarterclayton
-    - tallclair
-    - vikaschoudhary16
-    - vishh
-    - yguo0905
+    - SergeyKanzhelev
+    - sjenning
+    - wzshiming
     - yujuhong
     privacy: closed
   sig-node-pr-reviews:
     description: PR Reviews for Kubernetes Node Special-Interest-Group
     members:
-    - ConnorDoyle
+    - andrewsykim
+    - bobbypage
     - dchen1107
     - derekwaynecarr
-    - endocrimes
```

Comment:
Can you keep me on here? - Docs folks ping the alias and their PRs get somewhat ignored a lot otherwise.

Reply:
done